source/api-documentation.txt

```diff
@@ -9,7 +9,7 @@ API Documentation
    :maxdepth: 1
 
     BSON kotlinx.serialization <{+api-kotlin+}/apidocs/bson-kotlinx/index.html>
-    Core <{+api+}/apidocs/apidocs/mongodb-driver-core/index.html>
+    Core <{+api+}/apidocs/mongodb-driver-core/index.html>
```

Comment:
Suggestion:
From what I can tell, the `api` source constant still points to 4.9 -- did you intend it to point to 4.10 instead? I think those docs are published, so you should be able to link to them.

Reply:
I have the source constant updated as part of this PR: https://github.com/mongodb/docs-kotlin/pull/102 but I was hesitant to deploy changes today